mathsys/dev/builder/code.py

```
@@ -0,0 +1,91 @@
+    #~ CLASS -> FILENAME
+    def filename(self, target: str) -> str:
+        return os.path.join(
+            os.path.dirname(os.path.abspath(__file__)),
+            "..",
+            "target",
+            TARGETS[target],
+            "release",
+            f"mathsys{EXTENSIONS[target]}"
+        )
+    #~ CLASS -> EXECUTE
+    def execute(self, command: list[str], env: dict) -> None: subprocess.run(
```

Ruff (0.14.8)

`subprocess` call: check for execution of untrusted input

(S603)